icu4c/source/common/uvectr32.h

```diff
@@ -183,6 +183,11 @@ class U_COMMON_API UVector32 : public UObject {
      */
     virtual UClassID getDynamicClassID() const override;
 
+    /**
+     * Perform the quick sorting for the array elements between index low and index high.
+     */
+    void quickSort(int32_t low, int32_t high);
```

Comment:
Please call [`uprv_sortArray()`](https://github.com/unicode-org/icu/blob/main/icu4c/source/common/uarrsort.h) instead of implementing a second version of quick sort.

Reply:
Done. Remove it.